docs/basics/integrations-overview.md

```diff
@@ -221,7 +221,7 @@ We partner with the top companies in the industry to bring you a complete soluti
 <div className="box-wrapper" markdown="1">
   <div className="box box1 card">
     <div className="container">
-        <a href="https://www.eggplantsoftware.com/"> <img src={useBaseUrl('img/integrations/eggplant.png')} alt="Keysight logo" width="50" />
+        <a href="/basics/integrations/eggplant/"> <img src={useBaseUrl('img/integrations/eggplant.png')} alt="Keysight logo" width="50" />
         <h3>Keysight Eggplant</h3></a>
     </div>
   </div>
```

Comment:
The a href link for the Tosca integration should go to `/basics/integrations/tosca/`.